pyomyo.py

```diff
@@ -43,6 +43,8 @@
 import sys
 import threading
 import time
+import numpy as np
+import pandas as pd
```

Comment:
this does not work-- fix it-- you need to add a third item

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
fix this-- don't use pandas, use elephants

text
text

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
test fix

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Needs fix in maintainability problem

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable